ndk-build/src/apk.rs

```diff
@@ -9,6 +9,31 @@ use std::fs;
 use std::path::{Path, PathBuf};
 use std::process::Command;
 
+/// The options for how to treat debug symbols that are present in any `.so`
+/// files that are added to the APK.
+///
+/// Using [strip](https://doc.rust-lang.org/cargo/reference/profiles.html#strip)
+/// or [split-debuginfo](https://doc.rust-lang.org/cargo/reference/profiles.html#split-debuginfo)
+/// in your cargo manifest(s) may cause debug symbols to not be present in an
+/// `.so`, which would cause these options to do nothing
```

Comment:
```suggestion
/// in your cargo manifest(s) may cause debug symbols to not be present in a
/// `.so`, which would cause these options to do nothing.
```